package.json

```diff
@@ -35,7 +35,7 @@
     "textlint-scripts": "^12.1.1"
```

Comment:
Need to update `textlint-scripts`.
Run `yarn add textlint-scripts --dev`

Reply:
Fixed in 9e1d82822dbdc643a194f4c91ac075a93279d7a2